src/labels/CustomLabels.labels

```diff
@@ -4880,6 +4880,136 @@ To check your GAU Allocation settings, go to Donations | GAU Allocations.</value
         <shortDescription>Error completing the specified API call</shortDescription>
         <value>Error completing the {0} api: {1}</value>
     </labels>
+    <labels>
+        <fullName>batchNumberSettingsDescActivation</fullName>
+        <categories>Batch Number Settings</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Description of how to use the existing batch number format table</shortDescription>
+        <value>Activate or deactivate batch number formats. One batch number format can be active at time.</value>
+    </labels>
+    <labels>
+        <fullName>batchNumberSettingsHeaderFormats</fullName>
+        <categories>Batch Number Settings</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Header for table of existing batch number formats</shortDescription>
+        <value>Batch Number Formats</value>
+    </labels>
+    <labels>
+        <fullName>batchNumberSettingsSave</fullName>
+        <categories>Batch Number Settings</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Link to save a batch number format</shortDescription>
+        <value>Save Batch Number Format</value>
+    </labels>
+    <labels>
+        <fullName>batchNumberSettingsDescDisplayFormat</fullName>
+        <categories>Batch Number Settings</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Explanation of Display Format options</shortDescription>
+        <value>Display Format allows you to control the appearance of your auto-numbered
+            field. It must be unique and include the number of digits as well as a prefix, if
+            desired. Your format can be up to 30 characters.&lt;/br&gt;&lt;/br&gt;
```

Comment:
```suggestion
            desired. Your format can be up to 10 characters.&lt;/br&gt;&lt;/br&gt;
```

Reply:
What are we gaining by limiting to 10 characters?

Reply:
checks-00001 is 12 chars long.

Reply:
Not sure. I don't have strong feelings either way; I just want the custom label to be accurate. I thought limiting to 10 instead of 30 is what Ken mentioned during the demo yesterday.

Reply:
Tried to clarify and explain in this commit: let me know if this could still use some additional clarification and I'll work on editing the comment.

Reply:
Please ping me if you need me to do anything here. Bottom line, this custom label just needs to reflect the overall character limit including the number of digits and the prefix.

Reply:
I'm fine leaving this as "Your format can be up to 30 characters." Or removing the reference to length entirely.

Technically, the Display Format input field is limited to 23 char and the Starting Number is limited to 9 char.  

I'm going to leave this as is unless I hear otherwise!